docs/app/user-directory.md

```diff
@@ -2,54 +2,157 @@
 
 > [Users](/getting-started/glossary#users) are the individual accounts for logging in to the App. Each user belongs to a
 > [Role](/getting-started/glossary#roles) which defines its [Permissions](/getting-started/glossary#permissions). The
-> User Directory shows all users within the project, each with their own customizable profile page.
+> User Directory is the management system for all Users within a Project.
+
+![User Directory Page](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-directory-20220222A.webp)
 
 [[toc]]
 
-## Browsing Users
+## How it Works
+
+This module is a management system that enables one to View, Invite, Create, Edit, and Delete Users. In order to
+understand and use this module effectively, you will need to understand
+[Users, Roles, and Permissions](/configuration/users-roles-permissions.md).
+
+## Access Permissions
+
+The permissions configured for your Role will determine what you can see and do inside the User Directory.
+Administrators have full permissions. By default, Public has no access permissions _(to anything)_ and other _User_
+Roles have limited access permissions. These permissions can be fully configured in **Settings > Roles & Permissions**
+with all but one exception... A User must at the very least have read permissions for itself on the Directus Users
+Collection, so that is actually hard-wired and unchangeable in every new Role created. By default, new Roles also allow
+Users to Edit their own information, but this can be reconfigured.
```

Comment:
This only applies to roles with _App Access_ enabled. "regular" roles have no hardwired rules at all 👍🏻 

Maybe we can make it something like:

```suggestion
The permissions configured for your Role will determine what you can see and do inside the User Directory.
Administrators have full permissions. By default, Public has no access permissions _(to anything)_ and other _User_
Roles have access permissions based on their role. These permissions can be fully configured in **Settings > Roles & Permissions**. Roles with _App Access_ enabled have some limited permissions enabled by default, including the ability for users of that role to view their own user data. By default, new Roles created with App Access enabled also allow Users to Edit their own information, but this can be reconfigured at will.
```

(needs some copy writing magic)

Reply:
<img width="1728" alt="Screen Shot 2022-03-08 at 2 04 51 PM" src="https://user-images.githubusercontent.com/62306630/157317244-480defe6-7531-4198-8ae2-05d10f6c2177.png">

I was trying to address this issue above, but I think the whole topic needs to be addressed in the config docs.
I'm taking ur copy, slashing to less info and will address all this in the config docs.

Reply:
@rijkvanzanten Here is what I changed it to. With the plan to go deep in the corresponding config doc.


```suggestion
The permissions configured for your Role will determine what you can see and do inside the User Directory. Roles with
_Admin Access_ enabled are created with full Permissions. Roles with _App Access_ enabled are created with some limited
Permissions configured by default. Roles that have neither _Admin_ nor _App Access_ enabled (such as the built-in
_Public_ Role) are created with no access permissions _(to anything)_ by default. Default permissions can be fully
configured in **Settings > Roles & Permissions**.
```